controllers/index.js

```diff
@@ -9,13 +9,25 @@ exports.getEndpoints = (req, res, next) => {
 }
 
 exports.getTopics = (req, res, next) => {
-    return retrieveTopics()
+    return retrieveData('topics')
     .then(({rows}) => {
         res.status(200).send(rows)
     })
     .catch(next) 
 }
 
+exports.getArticlesById = (req, res, next) => {
+    const {params} = req
+    if (isNaN(params.id)) {
```

Comment:
what happens when we request with an invalid article id? does psql give us back something useful that means we dont have to do this check? 0.0